packages/charts/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -124,7 +124,7 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
     super(props);
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33460&commitId=b6b8b6e688f5daf1244928c91177ddf64bf4fbad&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Right center.chromium.png|2396| Changed|
|Callout.Top left edge.chromium.png|2307| Changed|
|Callout.Right top edge.chromium.png|1127| Changed|

</details>


  <details><summary><b>react-charting-HorizontalBarChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HorizontalBarChart.With_Axis.chromium.png|27107| Changed|
|react-charting-HorizontalBarChart.With Axis Dark Mode.chromium.png|8837| Changed|
|react-charting-HorizontalBarChart.With Axis RTL.chromium.png|27111| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>